addons/l10n_hu_edi/views/report_templates.xml

```diff
@@ -90,4 +90,69 @@
+            <div t-attf-class="footer o_company_#{company.id}_layout {{report_type != 'pdf' and 'position-relative mt-auto mx-n5'}}" t-if="custom_footer">
+                <t t-out="custom_footer"/>
+            </div>
+        </div>
+    </template>
+
+    <template id="external_layout_wave" inherit_id="web.external_layout_wave">
+        <!-- support for custom header -->
+        <div t-attf-class="header o_company_#{company.id}_layout {{report_type != 'pdf' and 'h-0'}}" position="attributes">
+            <attribute name="t-if">not custom_header</attribute>
+        </div>
+        <div t-attf-class="header o_company_#{company.id}_layout {{report_type != 'pdf' and 'h-0'}}" position="after">
+            <div t-attf-class="header o_company_#{company.id}_layout {{report_type != 'pdf' and 'h-0'}}" t-if="custom_header">
+                <t t-call="#{custom_header}"/>
+            </div>
+        </div>
+
+        <!-- support for custom footer -->
+        <div t-attf-class="footer o_company_#{company.id}_layout {{report_type != 'pdf' and 'position-relative mt-auto mx-n5'}}" position="attributes">
+            <attribute name="t-if">not custom_footer</attribute>
+        </div>
+        <div t-attf-class="footer o_company_#{company.id}_layout {{report_type != 'pdf' and 'position-relative mt-auto mx-n5'}}" position="after">
+            <div t-attf-class="footer o_company_#{company.id}_layout {{report_type != 'pdf' and 'position-relative mt-auto mx-n5'}}" t-if="custom_footer">
+                <t t-out="custom_footer"/>
+            </div>
+        </div>
+    </template>
+
+    <template id="external_layout_folder" inherit_id="web.external_layout_folder">
+        <!-- support for custom header -->
+        <div t-attf-class="header o_company_#{company.id}_layout {{report_type != 'pdf' and 'h-0'}}" position="attributes">
+            <attribute name="t-if">not custom_header</attribute>
+        </div>
+        <div t-attf-class="header o_company_#{company.id}_layout {{report_type != 'pdf' and 'h-0'}}" position="after">
+            <div t-attf-class="header o_company_#{company.id}_layout {{report_type != 'pdf' and 'h-0'}}" t-if="custom_header">
+                <t t-call="#{custom_header}"/>
+            </div>
+        </div>
+
+        <!-- support for custom footer -->
+        <div t-attf-class="footer o_company_#{company.id}_layout {{report_type != 'pdf' and 'mt-auto'}}" position="attributes">
+            <attribute name="t-if">not custom_footer</attribute>
+        </div>
+        <div t-attf-class="footer o_company_#{company.id}_layout {{report_type != 'pdf' and 'mt-auto'}}" position="after">
+            <div t-attf-class="footer o_company_#{company.id}_layout {{report_type != 'pdf' and 'mt-auto'}}" t-if="custom_footer">
+                <t t-out="custom_footer"/>
+            </div>
+        </div>
```

Comment:
repeated 4 times

Reply:
These ones are `<xpath>`, unsure there is a way to t-call them?